driver/src/main/com/mongodb/client/gridfs/GridFSBucket.java

```diff
@@ -166,6 +199,33 @@
     ObjectId uploadFromStream(String filename, InputStream source, GridFSUploadOptions options);
 
     /**
+     * Uploads a user file to a GridFS bucket.
```

Comment:
Change the wording on this to something like

     Uploads the contents of the given {@code InputStream} to a GridFS bucket

I find the use of the work file to be confusing, as there may never be a "file" per se.